src/main/java/seedu/revision/ui/StartQuizWindow.java

```diff
@@ -58,7 +59,24 @@
     private Answerable previousAnswerable;
     private Answerable currentAnswerable;
     private Iterator<Answerable> answerableIterator;
+    private int totalScore = 0; //accumulated score for completing all questions in entire quiz
+
+    //to keep track of current score for a particular level eg. level 2 -> 3 out of 5 questions correct
     private int score = 0;
+    //score of level 1 difficulty questions
+    private int score1 = 0;
+    //score of level 2 difficulty questions
+    private int score2 = 0;
+    //score of level 3 difficulty questions
+    private int score3 = 0;
+    //total number of questions answered in level 1
+    private int total1 = 0;
+    //total number of questions answered in level 2
+    private int total2 = 0;
+    //total number of questions answered in level 3
+    private int total3 = 0;
```

Comment:
Possible to store this in a dictionary format or an array? Doesn't look very nice code quality wise ah. Can consider that data structure to pass into your Statistics Class as well.

Reply:
ok will look into it